### PR TITLE
maintainers: add wiredhikari

### PR DESCRIPTION
**Pull request by wiredhikari**

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -13543,6 +13543,12 @@
     github = "wintrmvte";
     githubId = 41823252;
   };
+  wiredhikari = {
+    name = "WiredHikari";
+    email = "atharvaamritkar@protonmail.com";
+    githib = "wiredhikari";
+    githubID = 81508805;
+  }
```
Comment (dotlambda) [inline maintainers/maintainer-list.nix:13551]:
```suggestion
  };
```

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -13543,6 +13543,12 @@
     github = "wintrmvte";
     githubId = 41823252;
   };
+  wiredhikari = {
+    name = "WiredHikari";
```
Comment (dotlambda) [inline maintainers/maintainer-list.nix:13547]:
```suggestion
    name = "Atharva Amritkar";
```

File: maintainers/maintainer-list.nix [hunk 3]
```diff
@@ -13543,6 +13543,12 @@
     github = "wintrmvte";
     githubId = 41823252;
   };
+  wiredhikari = {
+    name = "Atharva Amritkar";
+    email = "atharvaamritkar@protonmail.com";
+    githib = "wiredhikari";
```
Comment (Mindavi) [inline maintainers/maintainer-list.nix:13549]: Typo

File: maintainers/maintainer-list.nix [hunk 4]
```diff
@@ -13543,6 +13543,12 @@
     github = "wintrmvte";
     githubId = 41823252;
   };
+  wiredhikari = {
+    name = "Atharva Amritkar";
+    email = "atharvaamritkar@protonmail.com";
+    github = "wiredhikari";
+    githubID = 81508805;
```
Comment (Mindavi) [inline maintainers/maintainer-list.nix:13550]: Sorry I missed this, this needs to be githubId

**Comment by Artturin** [comment]
this should be done in the same pr where you take maintainership of a package so we don't have unused maintainer entries

**Pull request closed without merging**